gui-libs/wlroots/wlroots-9999.ebuild

```diff
@@ -11,7 +11,7 @@ HOMEPAGE="https://gitlab.freedesktop.org/wlroots/wlroots"
 if [[ ${PV} == 9999 ]]; then
 	EGIT_REPO_URI="https://gitlab.freedesktop.org/${PN}/${PN}.git"
 	inherit git-r3
-	SLOT="0.18"
+	SLOT="9999"
```

Comment:
Isn't it going to collide with 0.18 when it's in-tree?

Reply:
No, at that moment it will be wlroots-0.19

Reply:
If they always branch then it should be fine, yeah. Or even if it's just a brief period it should be good enough. Duh. Thanks!

Reply:
I'm not sure I like this change. We should bump the SLOT based on upstream activity, cause with 9999, we don't cause rebuilds of consumers `:=`

If dwl-9999 needs to depend on wlroots-9999, it can just depend on `~wlroots-9999:=`.

Will I update the SLOT for wlroots-9999 always - probably no, since I don't follow that much upstream master activity. But when we notice an issue or something, just bump it :)